1.8/overview/maturity.md

```diff
@@ -0,0 +1,145 @@
+---
+post_title: Feature Maturity Phases
+nav_title: Feature Maturity
+menu_order: 10
+---
+
+The purpose of the Feature Maturity Phases is for Mesosphere Product and Engineering organizations to jointly educate customers, partners, and Mesosphere field and support organizations about the maturity and quality of features.
+
+- [Criteria](#criteria)
+- [Phases](#phases)
+- [Establishing Maturity](#establishing-maturity)
+- [Communication](#communication)
+- [Current DC/OS feature maturity](#current-maturity)
+
+# <a name="criteria"></a>Criteria
+
+## Completeness
+
+**Functionality:** Completeness of the feature implementation.
+
+**Interfaces:** Feature has an API with deprecation cycle, CLI, and UI.
```

Comment:
Strictly speaking not all features require a CLI and UI. We will not require CLI and UI for a feature to be considered mature.